### PR TITLE
Add testing for Login Component

### PR DESCRIPTION
**Pull request by TimNguyen21**
## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Refactoring- no new features
- [x] Testing

## Detailed Description
- Add testing for Login Component

## Why is this change required? What problem does it solve?
- Test if elements are rendering on the DOM properly

## Files modified:
- Login.js
- Login.test.js
- App.test.js

### CODE REVIEW

File: src/components/App/App.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { render } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  // const { getByText } = render(<App />);
+  // const linkElement = getByText(/learn react/i);
+  // expect(linkElement).toBeInTheDocument();
```
Comment (CLPena) [inline src/components/App/App.test.js:8]: If these are commented out, should we delete them?

File: src/components/Login/Login.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  //
+  //   const emailInputElement = getByPlaceholderText("enter email");
+  //   const passwordInputElement = getByPlaceholderText("enter password");
+  //
+  //   fireEvent.change(emailInputElement, {target: {value: "Yoo@email.com"}});
+  //   fireEvent.change(passwordInputElement, {target: {value: "password1"}});
+  //   fireEvent.click(getByText("Submit Login"));
+  // })
+
+})
```
Comment (CLPena) [inline src/components/Login/Login.test.js:57]: So thoughtful of you to include this for future features! Thanks!

File: src/components/Login/Login.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    const passwordInputElement = getByPlaceholderText("enter password");
+
+    fireEvent.change(emailInputElement, {target: {value: "Yoo@email.com"}});
+    fireEvent.change(passwordInputElement, {target: {value: "password1"}});
+
+    expect(emailInputElement.value).toBe("Yoo@email.com");
+    expect(passwordInputElement.value).toBe("password1");
+  })
+
+  //* test can be used in future when displaying bad credentials if worng login is used *//
```
Comment (megan-venetianer) [inline src/components/Login/Login.test.js:45]: Awesome job setting this up ahead of time!

**Comment by hborrelli1** [comment]
Nice job Tim, looks good.

**Pull request merged**